src/atoms/Tab/Tab.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-ondevice-knobs';
+import Tab from './Tab';
+import Flex from '../Flex';
+import View from '../View';
+
+storiesOf('Tabs', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    component: Tab,
+  })
+  .add('show tabs', () => (
```

Comment:
Let's keep app sub categories of stories as default
```suggestion
  .add('default', () => (
```